index.js

```diff
@@ -12,6 +13,10 @@ module.exports = function (numbers, opts) {
 	var finiteNumbers = numbers.filter(isFinite);
 	var min = opts.min || Math.min.apply(null, finiteNumbers);
 	var max = opts.max || Math.max.apply(null, finiteNumbers);
+	var color = opts.color || '';
+	if (color === 'fire') {
+		ticks = [colors.fg.getRgb(5,5,3) + '▁', colors.fg.getRgb(5,5,4) + '▂', colors.fg.getRgb(5,5,0) + '▃', colors.fg.getRgb(5,4,0) + '▄', colors.fg.getRgb(5,3,0) + '▅', colors.fg.getRgb(5,2,0) + '▆', colors.fg.getRgb(5,1,0) + '▇', colors.fg.getRgb(5,0,0) + '█'];
```

Comment:
I would also probably only define the colors in an array `[[5,5,3], [5,5,4], ...]` and then loop through it and apply the color function and the spark according to the index.